pkg/settings/settings.go

```diff
@@ -114,6 +138,11 @@ var (
 		NullTypeNative:    true,
 		NullTypePrimitive: true,
 	}
+
+	supportedFileNameFormats = map[FileNameFormat]bool{
```

Comment:
Also there is no need for a Go doc here, lets stick to the already documented maps and add some Go doc here as well 👍

Reply:
Added missing go doc!